ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -339,6 +346,7 @@ public StoreConfig(VerifiableProperties verifiableProperties) {
     storeMaxNumberOfEntriesToReturnFromJournal =
         verifiableProperties.getIntInRange("store.max.number.of.entries.to.return.from.journal", 5000, 1, 10000);
     storeDeletedMessageRetentionDays = verifiableProperties.getInt("store.deleted.message.retention.days", 7);
+    storeCompactionPolicySwitchPeriod = verifiableProperties.getInt("store.compaction.policy.switch.period", 7);
```

Comment:
let's set a limit here by using getInt(name, default, min, max). This would make sure that period would be 0 or negative number.

Reply:
Updated.